doc/reference/network_bridge.md

```diff
@@ -106,6 +106,7 @@ Key                                  | Type      | Condition             | Defau
 `ipv6.routing`                       | bool      | IPv6 address          | `true`                    | Whether to route traffic in and out of the bridge
 `maas.subnet.ipv4`                   | string    | IPv4 address          | -                         | MAAS IPv4 subnet to register instances in (when using `network` property on NIC)
 `maas.subnet.ipv6`                   | string    | IPv6 address          | -                         | MAAS IPv6 subnet to register instances in (when using `network` property on NIC)
+`queue.tx.length`                    | integer   | -                     | -                         | Sets the `txqueuelength` of the network device to the specified value.
```

Comment:
That seems wrong, the config key is a device config key, not a network config key.

So it should go in doc/reference/devices_nic.md instead, where it currently already is present.